ci_framework/roles/cert_manager/defaults/main.yml

```diff
@@ -0,0 +1,56 @@
+---
+# Copyright Red Hat, Inc.
+# All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License"); you may
+# not use this file except in compliance with the License. You may obtain
+# a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+# WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+# License for the specific language governing permissions and limitations
+# under the License.
+
+
+# All variables intended for modification should be placed in this file.
+# All variables within this role should have a prefix of "cifmw_cert_manager"
+
+cifmw_cert_manager_basedir: "{{ cifmw_basedir | default(ansible_user_dir ~ '/ci-framework-data') }}"
+cifmw_cert_manager_manifests_dir: "{{ cifmw_manifests | default(cifmw_cert_manager_basedir ~ '/artifacts/manifests') }}/cert-manager"
+cifmw_cert_manager_operator_namespace: cert-manager-operator
+cifmw_cert_manager_openshift_version: stable-v1
+
+cifmw_cert_manager_olm_operator_group:
+  apiVersion: operators.coreos.com/v1
+  kind: OperatorGroup
+  metadata:
+    annotations:
+      olm.providedAPIs: CertManager.v1alpha1.operator.openshift.io,Certificate.v1.cert-manager.io,CertificateRequest.v1.cert-manager.io,Challenge.v1.acme.cert-manager.io,ClusterIssuer.v1.cert-manager.io,Issuer.v1.cert-manager.io,Order.v1.acme.cert-manager.io
+    generateName: "{{ cifmw_cert_manager_operator_namespace }}-"
+    name: "{{ cifmw_cert_manager_operator_namespace }}-bccwx"
+    namespace: "{{ cifmw_cert_manager_operator_namespace }}"
+  spec:
+    targetNamespaces:
+      - "{{ cifmw_cert_manager_operator_namespace }}"
+    upgradeStrategy: Default
+
+cifmw_cert_manager_olm_subscription:
+  apiVersion: operators.coreos.com/v1alpha1
+  kind: Subscription
+  metadata:
+    labels:
+      operators.coreos.com/openshift-cert-manager-operator.cert-manager-operator: ""
+    name: openshift-cert-manager-operator
+    namespace: "{{ cifmw_cert_manager_operator_namespace }}"
+  spec:
+    channel: "{{ cifmw_cert_manager_openshift_version }}"
+    installPlanApproval: Automatic
+    name: openshift-cert-manager-operator
+    source: redhat-operators
+    sourceNamespace: openshift-marketplace
```

Comment:
I suggest to make these two customizable to allow installing from other sources.